docs/5.community/5.framework-contribution.md

```diff
@@ -59,13 +59,13 @@ Once you've read the [general contribution guide](/docs/community/contribution),
 
 While working on a PR, you will likely want to check if your changes are working correctly.
 
-You can modify the example app in `playground/`, and run it with `yarn dev`. Please make sure not to commit it to your branch, but it could be helpful to add some example code to your PR description. This can help reviewers and other Nuxt users understand the feature you've built in-depth.
+You can modify the example app in `playground/`, and run it with `npm run dev`. Please make sure not to commit it to your branch, but it could be helpful to add some example code to your PR description. This can help reviewers and other Nuxt users understand the feature you've built in-depth.
```

Comment:
```suggestion
You can modify the example app in `playground/`, and run it with `pnpm dev`. Please make sure not to commit it to your branch, but it could be helpful to add some example code to your PR description. This can help reviewers and other Nuxt users understand the feature you've built in-depth.
```